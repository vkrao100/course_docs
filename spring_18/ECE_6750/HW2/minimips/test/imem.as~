L: lw $v0, 0($at)
   lw $a0, 0($v1)
   add $a0, $a1, $a0
   add $a1, $v1, $a0
   sw $a0, 0($v1)
M: sub $a0,$a0,$v0
   beq $a0,$zero,L
   j M
