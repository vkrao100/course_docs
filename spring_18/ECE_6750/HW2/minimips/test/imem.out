8c220000
8c640000
00a42020
00642824
ac640000
00822022
1080fff9
08000005
